packages/opentelemetry-plugin-grpc/src/utils.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Equivalent to lodash _.findIndex
```

Comment:
Thanks for writing this utility rather than pulling in lodash itself! (I have seen vulnerabilities on it recently and wouldn't want to expose OpenTelemetry customers to that risk)